Formula/j/jsontoolkit.rb

```diff
@@ -0,0 +1,52 @@
+class Jsontoolkit < Formula
+  desc "Swiss-army knife library for expressive JSON programming in modern C++"
+  homepage "https://jsontoolkit.sourcemeta.com/"
+  url "https://github.com/sourcemeta/jsontoolkit/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "00f82f02166beabec80522e2bbc7b839ee9b7ccb631411c42e6fab65186e80ba"
+  license "AGPL-3.0-only"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %W[
+      -S .
+      -B build
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF
+      -DJSONTOOLKIT_CONTRIB=OFF
+      -DJSONTOOLKIT_TESTS=OFF
+      -DJSONTOOLKIT_DOCS=OFF
+      -DBUILD_SHARED_LIBS=OFF
+    ]
+
+    system "cmake", *args
+    system "cmake", "--build", "build", "--config", "Release", "--parallel"
```

Comment:
```suggestion
    system "cmake", "--build", "build"
```
You don't need this.